source.py

```diff
@@ -2,8 +2,11 @@
 from time import sleep
 from PIL import Image
 import cv2
-
-
+import RPi.GPIO as GPIO
+import time # time.time() is also needed to calc distance with ultrasonic sensors
+import xbox #assuming xbox will be in different file
+# edited rangesensor class
+GPIO.setmode(GPIO.BOARD) # I assume this is something we want to set at the top?
```

Comment:
This means whenever we look at any of the pins the number for which pin is the actual number we type in